tests/test_overfocus.py

```diff
@@ -60,23 +60,23 @@ def test_get_transformation_matrix(params):
                 'y_px': 3.,
                 'x_px': -7.,
             },
-            (3, -7)
+            (1.5, -3.5)
         ),
         (
             {
                 'y_px': 3.,
                 'x_px': -7.,
                 'transformation_matrix': np.array(((-1., 0.), (0., -1.))),
             },
-            (-3, 7)
+            (-1.5, 3.5)
         ),
         (
             {
                 'detector_pixel_size': 2,
                 'y_px': 3.,
                 'x_px': -7.,
             },
-            (6, -14)
+            (3, -7)
```

Comment:
Where does the difference in factor 2 come from in these? Is that because of `camera_length + overfocus` where previously it was factor 1 from `camera_length` including the `overfocus`? Or from the `detector_pixel_size` in some way? Might need a comment.

The "default parameter" thing makes this also a bit hard to understand, as one needs to take into account things that are not mentioned in the parameters we can see here in the diff. Including the relevant parameters in each parameter set would be helpful.